pkg/api/common/const.go

```diff
@@ -84,13 +84,13 @@ var AllDCOSSupportedVersions = []string{
 }
 
 const (
-	// OpenShiftVersion3Dot9Dot0 is the major.minor.patch string for the 3.9.0 version of OpenShift
-	OpenShiftVersion3Dot9Dot0 string = "3.9.0"
+	// OpenShiftVersion3Dot9Dot14 is the major.minor.patch string for the 3.9.14 version of OpenShift
+	OpenShiftVersion3Dot9Dot14 string = "3.9.14"
```

Comment:
@jim-minter this depends on the binaries installed in the node image. Unfortunately it seems we have different patch versions between rhel and centos so the version check inside the e2e tests fails. Any idea what should we do (eg. we could ignore the patch version in the test?) and whether tracking the correct patch version here makes sense or not?

cc @kwoodson